pkg/dotc1z/c1file.go

```diff
@@ -67,9 +75,13 @@ func NewC1File(ctx context.Context, dbFilePath string, opts ...C1FOption) (*C1Fi
 	db := goqu.New("sqlite3", rawDB)
 
 	c1File := &C1File{
-		rawDb:      rawDB,
-		db:         db,
-		dbFilePath: dbFilePath,
+		rawDb:                 rawDB,
+		db:                    db,
+		dbFilePath:            dbFilePath,
+		pragmas:               []pragma{},
+		slowQueryLogTimes:     make(map[string]time.Time),
+		slowQueryThreshold:    5 * time.Second,
```

Comment:
Thoughts on the values?

Reply:
I think 5 seconds is fine for now.

Can we limit the maximum number of items in slowQueryLogTimes? A sync does a lot of different queries and I don't want to keep a ton of them around. Also I think we should only log this once a minute maximum, not once a minute per unique query.

Reply:
The query that we key on is fully escaped/uninterpolated.  How many entries do you expect in that case?